packages/components/src/components/symbol/shadow.tsx

```diff
@@ -7,7 +7,7 @@ import { API, States } from './types';
 
 @Component({
 	tag: 'kol-symbol',
-	shadow: false,
+	shadow: true,
```

Comment:
@deleonio Ich weiß nicht, warum das zuvor keine Shadow Komponente war. Siehst du irgendwelche Probleme mit dieser Änderung?

Wir haben auch keinen Testfall in der React App für Symbol. Sollten wir einen hinzufügen oder ist Komponente nur für die interne Nutzung gedacht?

Reply:
Die Symbol-Komponente war für das Vorlesen von Sonderzeichen gedacht. Ich würde sagen, ja. Da der Span bestenfalls die Formatierung von außen übernehmen sollte.

Dann die Implementierung lieber in SymbolWc (`shadow: false`) umziehen. Symbol (`shadow: true`) wie bei den anderen als Wrapper Component.